### ISSUE TITLE
Passing options doesn't seem to work

### ISSUE THREAD

**Issue by dkarlovi** [robertfausk/behat-panther-extension#2]
When trying to pass options to Panther, it currently doesn't seem possible, I get

```
  [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]      
  Unrecognized option "options" under "testwork.panther". Available options are "".
```

Sample config

```yaml
# behat.yaml.dist
default:
  extensions:
    Robertfausk\Behat\PantherExtension:
      options:
        browser: chrome
```

**Comment by robertfausk** [comment]
Hi @dkarlovi! You can now configure as follows. Hope this helps!
```YAML
    # in behat.yml
        extensions:
            Robertfausk\Behat\PantherExtension: ~ # no configuration here
            Behat\MinkExtension:
               javascript_session: javascript_chrome
               sessions:
                   default:
                       panther: ~
                   javascript:
                       panther:
                           options: ~
                   javascript_chrome:
                       panther:
                           options:
                               browser: 'chrome'
                               webServerDir: '%paths.base%/public' # your custom public dir
                   javascript_firefox:
                       panther:
                           options:
                               browser: 'firefox'
    ```

**Comment by robertfausk** [comment]
Just released ```v1.0.1```. Please comment this issue again if there are some problems.
